scripts/deploy-sites-docker.sh

```diff
@@ -2,5 +2,5 @@
 
 TARGET=${1:-pointnetwork_website_owner}
 
-docker exec -it $TARGET /app/scripts/deploy-sites.sh /data "$@"
+docker exec -it $TARGET /app/scripts/deploy-sites.sh
```

Comment:
Should we leave the `"$@"` argument passing in order to be able to pass options like `--contracts` and `-v` to the `scripts/deploy-sites.sh` script inside container?